app/src/main/java/de/lilithwittmann/voicepitchanalyzer/models/database/RecordingDB.java

```diff
@@ -352,12 +349,22 @@ public String getName() {
             return name;
         }
 
-        public String getType() {
+        public ColumnType getType() {
             return type;
         }
 
         public String getDefinition() {
-            return name + type;
+            return name + " " + type.toSql();
+        }
+    }
+
+    private enum ColumnType {
+        TEXT,
+        DOUBLE,
```

Comment:
Hmm, I just noticed that, above, `DOUBLE_TYPE` was defined as `" REAL"` instead of `" DOUBLE"` but the two seem to be equivalent in SQLite.. and `DOUBLE` seems to convey the intent better anyway..